plugins/lookup/passwordstore.py

```diff
@@ -60,19 +60,46 @@
         description: use alphanumeric characters.
         type: bool
         default: 'no'
+      missing:
+        description: List of preference about what to do if the password file is missing.
```

Comment:
```suggestion
        description:
          - List of preference about what to do if the password file is missing.
          - If I(create=true), the value for this option is ignored and assumed to be C(create).
```

Reply:
Good point. I will include that to prevent confusion.